open-metadata-implementation/repository-services/repository-services-apis/src/main/java/org/odpi/openmetadata/repositoryservices/connectors/stores/metadatacollectionstore/OMRSMetadataCollectionBase.java

```diff
@@ -5868,7 +5869,7 @@ public  void deleteRelationshipReferenceCopy(String         userId,
         /*
          * Delete relationship
          */
-        reportUnsupportedOptionalFunction(methodName);
+        super.deleteRelationshipReferenceCopy(userId, relationship);
```

Comment:
@grahamwallis these lines where we have replaced the `reportUnsupportedOptionalFunction()` method call with a super-class call to the method itself look suspect to me:

- will this mean repositories that have not overridden these methods will no longer automatically throw the FNS exception?
- why have we replaced these calls only for the reference copy operations, and none of the others?